frontend/src/libs/components/stepper/stepper.tsx

```diff
@@ -0,0 +1,103 @@
+import { Button } from '~/libs/components/components.js';
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import { useCallback, useMemo, useState } from '~/libs/hooks/hooks.js';
+
+import { ReactComponent as Arrow } from '../../../assets/img/arrow-left.svg';
```

Comment:
```suggestion
```
create `Icom` component. It should receive icon type as string and return an appropriate icon